emuu-client/src/components/Sidebar/Sidebar.js

```diff
@@ -102,29 +96,74 @@ const Drawer = styled(MuiDrawer, {
   }),
 }));
 
-export default function MiniDrawer({ sideBarState }) {
+export default function MiniDrawer() {
   const theme = useTheme();
+  const [updatedSubscribersList, setUpdateSubscribersList] = useState([]);
+  const [
+    updatedSubscribersListCompleteData,
+    setUpdateSubscribersListCompleteData,
+  ] = useState([]);
+  const [users, setUsers] = useState([]);
   const history = useHistory();
-  const [ProfilePic,setProfilePic]=useState('');
-  const firebaseData = JSON.parse(localStorage.getItem("firebase-data"));
-  const usersArr = firebaseData.filter(
-    (obj) => obj.hasOwnProperty("Username") && !obj.hasOwnProperty("VideoUrl")
-  );
   const authUsersNavigation = ["Home", "UserProfile", "UploadVideo"];
   const unAuthorizedNavigation = ["Home"];
   const isAuthorized = localStorage.getItem("auth");
+  const userName = localStorage.getItem("displayName");
   const currentNavigation =
     isAuthorized === "true" ? authUsersNavigation : unAuthorizedNavigation;
-  const userProfileImg = localStorage.getItem("userProfileImg");
-    const displayName = localStorage.getItem("displayName");
-    const docRef = doc(db, "Users", displayName);
-
-    getDoc(docRef).then((docSnap) => {
-      setProfilePic(docSnap.data().ProfilePictureUrl);
-    });
-    console.log(ProfilePic);
+  const firebaseData = JSON.parse(localStorage.getItem("firebase-data"));
```

Comment:
Why store firebase data in local storage and then pull from that? We can pull from Firebase from the backend and send that to the frontend.